### DISCUSSION TITLE
How does an individual tick off a common task list?

### DISCUSSION THREAD

**Discussion by accesstechnology-mike** [outline/outline#2795] [Self Hosting]
I've not figured out how to achieve this well - I have a list on Outline for employee onboarding, that each new starter needs to run through.

What's a sensible workflow for this? Currently, they're copying it into a Google doc and then marking the individual items complete, which ruins formatting and is a horrible workaround.

Is there a method for individuals to run through a task list?

**Comment by tommoor** [comment]
I'd probably recommend to convert the checklist into a template, you can then easily create a new checklist for each employee as they start. You could take it one step further and also create a collection for each new employee that contains the checklist and other docs personal to them.

**Comment by tommoor** [comment]
I'd probably recommend to convert the checklist into a template, you can then easily create a new checklist for each employee as they start. You could take it one step further and also create a collection for each new employee that contains the checklist and other docs personal to them.
